### PR TITLE
#161342794 Update app to run test on travisCI

### PR DESCRIPTION
**Pull request by TheoOkafor**
#### What does this PR do?
- Update app to run the test on TravisCI
- Configure 'npm start' to transpile app to ES5 and run it.
- Refactor some migration and model files.
- Create a sample test file.

#### Description of Task to be completed?
- Test the configuration on a working route.
- Integrate code coverage

#### How should this be manually tested?
- git clone `https://github.com/andela/haven-ah-backend.git `
- run `npm install` to install the npm packages
- run `npm test` to run the sample test using mocha.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161342794

#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,23 +21,26 @@
...
     "pg-hstore": "^2.3.2",
-    "sequelize": "^4.41.0",
-    "slug": "^0.9.1",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-plugin-add-module-exports": "^1.0.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-stage-2": "^6.24.1",
-    "nodemon": "^1.18.4"
+    "slug": "^0.9.1"
```
Comment (seyi-adeleke) [inline package.json]: Slug is repeated, pg and pg-hstore shouldn't be in dev depedencies

File: server/app.js [hunk 2]
```diff
@@ -16,19 +16,19 @@ app.use(logger('dev'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-//app.use('/api/v1', routes);
+// app.use('/api/v1', routes);
```
Comment (seyi-adeleke) [inline server/app.js]: remove all commented out code

File: server/models/index.js [hunk 3]
```diff
@@ -1,28 +1,28 @@
 import fs from 'fs';
 import path from 'path';
 import Sequelize from 'sequelize';
+import config from '../config/config';
 
 const basename = path.basename(module.filename);
 const env = process.env.NODE_ENV || 'development';
-const config = require(`${__dirname}/../config/config.js`)[env];
+const configObj = config[env];
```
Comment (seyi-adeleke) [inline server/models/index.js:8]: 👍

**Comment by seyi-adeleke** [comment]
Let's keep the app.js file at the root directory

File: server/app.js (hunk 2 above)
Comment (TheoOkafor) [reply server/app.js]: Okay

File: server/models/index.js (hunk 3 above)
Comment (TheoOkafor) [reply server/models/index.js:8]: Priscilla did that.

File: package.json (hunk 1 above)
Comment (TheoOkafor) [reply package.json]: Okay

**Review by seyi-adeleke — APPROVED** [review]

**Pull request merged**